cmd/runfn/runfn.go

```diff
@@ -0,0 +1,217 @@
+package runfn
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/konveyor/crane/internal/flags"
+	"io"
+	"sigs.k8s.io/kustomize/kyaml/yaml"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"sigs.k8s.io/kustomize/cmd/config/runner"
+	"sigs.k8s.io/kustomize/kyaml/errors"
+	"sigs.k8s.io/kustomize/kyaml/fn/runtime/runtimeutil"
+	"sigs.k8s.io/kustomize/kyaml/runfn"
+)
+
+type Options struct {
+	// cobraGlobalFlags for explicit CLI args parsed by cobra
+	cobraGlobalFlags *flags.GlobalFlags
+	Flags
+}
+
+type Flags struct {
+	ExportDir          string
+	TransformDir       string
+	Image              string
+	Env                []string
+	RunFns             runfn.RunFns
+	TransformedContent bytes.Buffer
+}
+
+func NewFnRunCommand(f *flags.GlobalFlags) *cobra.Command {
+	o := &Options{
+		cobraGlobalFlags: f,
+	}
+	cmd := &cobra.Command{
+		Use:          "runfn [IMAGE] [flags] [--args]",
+		Short:        "Transform resources by executing function \nExperimental: This command is under active development and may change without notice.",
```

Comment:
```suggestion
		Short:        "Transform resources by executing KRM function \n\nExperimental: This command is under active development and may change without notice.",
```

Reply:
1. Done!
2. Added `Long`  command message, as the short message appears in `crane -v` and `crane -h`